jx_sqlite/query_table.py

```diff
@@ -276,7 +281,10 @@ def query(self, query):
                 row = [None for _ in column_names]
                 for s in index_to_columns.values():
                     if s.push_child == ".":
-                        row[s.push_column] = s.pull(d)
+                        if not row[s.push_column]:                        
+                            row[s.push_column] = s.pull(d)
+                        else:
+                            row[s.push_column] += s.pull(d)
```

Comment:
Instead of adding the columns using Python; Alter the SQL query to do the add for you:  Instead of 

               value=s.value.var
               columns=[c.es_column for c in self.sf.columns if untyped_column(c.es_column)[0]==value]
               for sql in columns:
                   column_number = len(outer_selects)
                   sql = "COUNT(" + quote_table(sql) + ")"

do something like 

               value=s.value.var
               columns=[c.es_column for c in self.sf.columns if untyped_column(c.es_column)[0]==value]
               column_number = len(outer_selects)
               sql = " + ".join("COUNT(" + quote_table(col) + ")" for cols in columns)

then you have only one column in your resultset which adds all the columns